src/lib/lib/frameloop.ts

```diff
@@ -53,17 +51,21 @@ export const useFrameloop = (
   renderCtx: ThrelteRenderContext
 ): void => {
   useRaf(() => {
+    if (renderCtx.frameInvalidated || ctx.pointerInvalidated) {
+      console.log("raycast")
+      animationFrameRaycast(ctx, rootCtx)
+      ctx.pointerInvalidated = false
+    }
     if (
       renderCtx.frameloop === 'demand' &&
       !renderCtx.frameInvalidated &&
-      rootCtx.interactiveObjects.size === 0 &&
       renderCtx.frameHandlers.size === 0
     ) {
       return
     }
     const camera = get(ctx.camera)
     if (!camera || !ctx.composer || !ctx.renderer) return
-    animationFrameRaycast(ctx, rootCtx)
+    console.log("render")
```

Comment:
I always use this tiny handy function to log to an expression that you can realtime track in Chrome DevTools without your console exploding:
```ts
import { browser } from '$app/env'

export const logToExpression = (name: string, value: any) => {
	if (!browser) return
	;(window as any)[name] = value
}
```

<img width="516" alt="Bildschirmfoto 2022-04-21 um 10 18 50" src="https://user-images.githubusercontent.com/46897060/164411699-c0b14367-02ba-45d2-befb-73e40f368ee1.png">